### PR TITLE
GEOMESA-1881 Accumulo option for storing timestamps instead of logica…

### PR DESCRIPTION
**Pull request by elahrvivaz**
…l time

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: docs/user/accumulo/data_management.rst [hunk 1]
```diff
@@ -42,6 +42,129 @@ be answered without joining against the record table. This is the only option fo
...
+- ``org.locationtech.geomesa.index.conf.DigitSplitter``
+
+  Assumes an even distribution of IDs starting with numeric values, which are specified as options
+
+Custom splitters may also be used - any class that extends ``org.locationtech.geomesa.index.conf.TableSplitter``.
+
+Specifying a Table Splitter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Table splitter may be specified by setting a hint when creating a simple feature type,
```
Comment (jnh5y) [inline docs/user/accumulo/data_management.rst]: Ok, what does a table splitter do?  This section could have a one or two about why I'd consider using one.

File: docs/user/accumulo/data_management.rst [hunk 2]
```diff
@@ -158,110 +281,3 @@ only populate features matching a CQL filter (e.g. the last month), or choose to
...
-
-Splitting the Record Index
---------------------------
-
-By default, GeoMesa assumes that feature IDs are UUIDs, and have an even distribution. If your
-feature IDs do not follow this pattern, you may define a custom table splitter for the record index.
-This will ensure that your features are spread across several different tablet servers, speeding
-up ingestion and queries.
-
-GeoMesa supplies three different table splitter options:
```
Comment (jnh5y) [inline docs/user/accumulo/data_management.rst]: Ah!  Did this just get straight deleted:)?

Comment (jahhulbert-ccri) [reply docs/user/accumulo/data_management.rst]: moved?!? ...

Comment (elahrvivaz) [reply docs/user/accumulo/data_management.rst]: this diff is highly misleading ;) I just added the section on 'accumulo logical timestamps'

File: docs/user/accumulo/data_management.rst (hunk 1 above)
Comment (elahrvivaz) [reply docs/user/accumulo/data_management.rst]: see other comment, i didn't actually modify this part of the docs (although your question is reasonable :)

Comment (jahhulbert-ccri) [reply docs/user/accumulo/data_management.rst]: push it to a new ticket if we want a doc review for next time. this is just a move

**Comment by elahrvivaz** [comment]
i'm going to add some checks on table sharing, etc to ensure logical time is actually applied correctly

**Pull request merged**